### PR TITLE
Better handling of default xml namespace

### LINKED ISSUE #50: Built in `xml` prefix importing leads to warnings
__Describe the Bug__

The `xml` prefix is built into XML and does not have to be declared explicitly (cf. [W3 namespace spec](https://www.w3.org/XML/1998/namespace).

The following is a valid XML document that should import without warnings:

```xml
<props:root xml:lang="en" />
```

__Steps to Reproduce__

* Open diagram
* __See import warnings__

__Expected Behavior__

* [ ] Diagram with `xml:lang` attributes or any other [valid `xml:...` attribute](https://www.w3.org/XML/1998/namespace) should not be reported as warnings.
* [ ] Attributes such as `xml:lang` or any other [valid `xml:...` attribute](https://www.w3.org/XML/1998/namespace) should be serialized upon roundtrip.

__Environment__

- Host (Browser/Node version), if applicable: Any
- OS: Any
- Library version: 9.0.2, 8.0.7

-------

Related to https://github.com/nikku/saxen/issues/5.

### PR DESCRIPTION
**Pull request by onlineandy**
Fixes #50

### CODE REVIEW

File: lib/common.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ export function hasLowerCaseAlias(pkg) {
 }
 
 export var DEFAULT_NS_MAP = {
-  'xsi': 'http://www.w3.org/2001/XMLSchema-instance'
+  'xsi': 'http://www.w3.org/2001/XMLSchema-instance',
+  'xml': 'http://www.w3.org/XML/1998/namespace',
```
Comment (nikku) [inline lib/common.js:7]: Drop trailing comma.

File: lib/read.js [hunk 2]
```diff
@@ -824,7 +824,8 @@ Reader.prototype.fromXML = function(xml, options, done) {
 
     return uriMap;
   }, {});
-
+  // add default xml ns
+  uriMap['http://www.w3.org/XML/1998/namespace'] = 'xml';
```
Comment (nikku) [inline lib/read.js:828]: Can we initialize the XML mapping on line 826, i.e. feed it as the default mapping into reduce:

```javascript
var uriMap = model.getPackages().reduce(function(uriMap, p) {
    uriMap[p.uri] = p.prefix;

    return uriMap;
  }, {
    'http://www.w3.org/XML/1998/namespace': 'xml'
  });
```

File: test/spec/reader.js [hunk 3]
```diff
@@ -127,6 +127,27 @@ describe('Reader', function() {
         });
       });
```
Comment (nikku) [inline test/spec/reader.js:129]: Please add another space before spec. We use double spaces between specs.

File: lib/common.js (hunk 1 above)
Comment (onlineandy) [reply lib/common.js]: done

**Comment by nikku** [comment]
Thanks :clap:

**Pull request merged**